src/System.Diagnostics.Debug/tests/DebugTests.cs

```diff
@@ -9,6 +9,7 @@ namespace System.Diagnostics.Tests
     public class DebugTests
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Net46)]
         public void Asserts()
         {
             VerifyLogged(() => { Debug.Assert(true); }, "");
```

Comment:
Will file a follow up issue on this.. Currently have other failures to unblock, the lib owner should be able to do this..